ci-operator/config/openshift/aws-ebs-csi-driver/openshift-aws-ebs-csi-driver-master.yaml

```diff
@@ -19,7 +19,18 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13
   to: aws-ebs-csi-driver
+- dockerfile_literal: |-
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13 AS builder
+    WORKDIR /go/src/k8s.io/kubernetes
+    RUN git clone --depth 1 https://github.com/openshift/kubernetes.git .
+    RUN make WHAT=k8s.io/kubernetes/openshift-hack/commitchecker
+    FROM registry.ci.openshift.org/ocp/4.13:base
+    COPY --from=builder /go/src/k8s.io/kubernetes/_output/bin/commitchecker /bin/commitchecker
+  from: base
+  to: commitchecker
```

Comment:
This is a quick & dirty hack to compile `commitchecker` and publish it in `commitchecker` image. The end goal is to move commitchecker out of github.com/openshift/kubernetes + have a proper image with it.